src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/tabs/signature_approved.jelly

```diff
@@ -0,0 +1,70 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
+    <div class="custom-pane-frame-padding">
+        <div>Signatures already approved:</div>
```

Comment:
Why is some of this not localizable?

Reply:
No changes applied in signature / class path parts except code "moved" into tabs layout